osc/core.py

```diff
@@ -235,6 +235,13 @@ def cmp(a, b):
 Link info: %s
 """
 
+project_info_templ = """\
+Project name: %s
+Path: %s
+API URL: %s
+Source URL: %s
+"""
+
```

Comment:
It might be better to use KeyValueTable like in this case: https://github.com/openSUSE/osc/blob/master/osc/obs_api/keyinfo_pubkey.py#L39

Reply:
I copied this part from package info_templ - should that switch then as well?

Reply:
Let's keep it this way. I'll eventually change it as part of a bigger refactoring.